bazel/repositories.bzl

```diff
@@ -0,0 +1,86 @@
+# The build rules below for external dependencies build rules are maintained on a best effort basis.
+# The rules are provided for developer convenience. For production builds, we recommend building the
+# libraries according to their canonical build systems and expressing the dependencies in a manner
+# similar to ci/WORKSPACE.
+
+def googletest_repositories():
+    BUILD = """
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
This is the only file ("generated" or not) that has the copyright. We should either drop it here (we can, since it's ours) or include in every `BUILD` file.

FWIW, OSPO usually requires copyright / license header in _every_ single file, but it looks that's not the case in Envoy.